lib/foreman_inventory_upload/generators/metadata.rb

```diff
@@ -28,6 +29,9 @@ def render_report(metadata)
           @stream.simple_field('report_id', Foreman.uuid)
           @stream.simple_field('host_inventory_api_version', '1.0')
           @stream.simple_field('source', 'Satellite')
+          @stream.simple_field('reporting_host_name', fqdn(ForemanRhCloud.foreman_host)) unless Setting[:obfuscate_inventory_hostnames]
+          @stream.simple_field('reporting_host_ips', host_ips(ForemanRhCloud.foreman_host).values.join(',')) unless Setting[:obfuscate_inventory_ips]
```

Comment:
Both `fqdn` and `host_ips` methods are aware of the obfuscation. And since the obfuscation algorithm is known, there is actual value even if we send the obfuscated data.
We could ask the user to run obfuscation for all his host names and match to the one that is in this report, so the user will be able to locate the host.

Reply:
@ShimShtein I am not sure what that change would look like, can you point to an example?